ts/src/index.ts

```diff
@@ -84,6 +84,15 @@ function initConfigLocal(config: Config): ProfilerConfig {
   const mergedConfig =
       extend(true, {}, defaultConfig, envSetConfig, envConfig, config);
 
+  // If sourceMapSearchPath is empty array in config, extend will override
+  // its value with the default value in mergedConfig (as if
+  // sourceMapSearchPath were not specified in config), rather than keep the
+  // empty array as sourceMapSearchPath's value in mergedConfig.
+  if (Array.isArray(config.sourceMapSearchPath) &&
```

Comment:
I see it now, thanks for the updated comment. So it's basically about distinguishing undefined sourceMapSearchPath vs. empty array sourceMapSearchPath values and hacking things up so that the latter can be used as an override rather then be ignored as in the current extend() logic.

What's the use case for specifying empty sourceMapSearchPath array though? Is this to disable source maps support? But there is disableSourceMaps option for that. Having two ways for that seems redundant, if that's the intention.

Reply:
Yeah -- it would be redundant. 
`disableSourceMaps=true` is better (certain functions won't be called).
But I also hate the idea that a user could pass in an empty array and it would be replaced with the default value, and source maps would be found and used. 

So I'm slightly for keeping this, but it's not a strong feeling.

Reply:
I think you should throw an error if the user passes an empty `sourceMapSearchPath` and advise users to use `disableSourceMaps` instead.

Reply:
Done!
That's a much better solution.